tests/db/aliasmappertest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * @author Tahaa Karim <tahaalibra@gmail.com>
+ *
+ * ownCloud - Mail
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+namespace OCA\Mail\Db;
+
+use OC\AppFramework\Db\Db;
+
+class AliasMapperTest extends \PHPUnit_Framework_TestCase {
+
+	/**
+	 * @var AliasMapper
+	 */
+	private $mapper;
+
+	/**
+	 * @var \OC\DB\Connection
+	 */
+	private $db;
+
+	/**
+	 * @var Alias
+	 */
+	private $alias;
+
+	/**
+	 * Initialize Mapper
+	 */
+	public function setup(){
```

Comment:
would be great to have a `tearDown` method too, which restores the previous db state